csrc/scheduler/matmul.cpp

```diff
@@ -1071,6 +1142,9 @@ void scheduleMatmul(Fusion* fusion, const MatmulParams& params) {
             .propagateParallelType()
             .propagateToBoundary());
     smem_epilogue->axis(-1)->parallelize(ParallelType::Vectorize);
+    if (num_splitk_dims != 0) {
+      splitk_sum->axis(-3)->parallelize(ParallelType::BIDz);
```

Comment:
Could you move all the scheduling of `splitk_sum` into a single utility function? Also, I don't understand what this line is trying to do. Is it parallelizing the `iNw` as `BIDz`?

Reply:
I refactored this PR to place all the scheduling of splitk_sum in a single function. That was a great idea as it showed me a few ways to simplify things. I think it's clearer now.